### PR TITLE
windows installer hacks

### PR DESCRIPTION
**Pull request by cgcostume**
fix for windows deployment + add `ENABLE_WIN_INSTALL_HACKS` flag to cmake scripts for windows installer - i is still required to merge the final examples and tools archives manually ... 😞

### CODE REVIEW

**Comment by cgcostume** [comment]
from now on, it should keep the windows deployment at least on the same level as linux deployment. There should be no release without thorough windows and linux testing and we should provide pre-compiled archives for at least every major and minor release. For now, windows is still the major platform for graphics development!

**Comment by cgcostume** [comment]
@scheibel - please check if everything still works on linux so i can release 3.0.3 asap.

**Comment by scheibel** [comment]
Works on my native Ubuntu 18.04.

File: source/glbinding/include/glbinding/gl/types.h [hunk 1]
```diff
@@ -1,1586 +1,1586 @@
...
    return a > static_cast<std::underlying_type<GLenum>::type>(b);
}

GLBINDING_CONSTEXPR inline bool operator>=(std::underlying_type<GLenum>::type a, const GLenum & b)
{
    return a >= static_cast<std::underlying_type<GLenum>::type>(b);
}


} // namespace gl
```
Comment (scheibel) [inline source/glbinding/include/glbinding/gl/types.h]: Please don't merge now; I first want to get rid of those stray carriage returns.

**Pull request merged**